packages/block-editor/src/components/ungroup-button/index.native.js

```diff
@@ -6,11 +6,11 @@ import { ToolbarGroup, ToolbarButton } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
+import { Ungroup } from '@wordpress/icons';
```

Comment:
@t-hamano I noticed when testing that the icon wasn't being rendered. I checked [the icons index file](https://github.com/WordPress/gutenberg/blob/bc7030daa86543d6d9301bb9844caa09de36c4b8/packages/icons/src/index.js#LL258C21-L258C28) and all are exported in lowercase, hence we'd need to update the import statement:
```suggestion
import { ungroup } from '@wordpress/icons';
```
<img src=https://github.com/WordPress/gutenberg/assets/14905380/bd40c88a-b75d-4220-870b-b516f9705563 width=300>

Reply:
@fluiddot 
Thank you for testing it. I fixed it with ce87ef7.